docs/docs/command-line-interface.mdx

```diff
@@ -585,6 +586,25 @@ key is missing or if the default value has not been changed.
 If you pass a `max_history` value to one or more policies in your `config.yml` file, provide the
 smallest of those values in the validator command using the `--max-history <max_history>` flag.
 
+### Validate flows
+
+The validator will perform the following checks on flows:
+- determine whether flow names or descriptions are unique after stripping punctuation
+- verify whether natural language conditions used in branch steps or `collect` step rejections are valid `pypred` expressions
```

Comment:
`pypred` is not a well know dependency, so I think we need to explain the predicates and how they work (and probably link to pypred repo?)

Reply:
or the alternative is to remove "`pypred`" from the sentence

Reply:
We mention pypred predicates in various places regarding [slot validation](https://12795--rasahq-docs-rasa-v2.netlify.app/docs/rasa/concepts/flows#slot-validation) and [branching logic](https://12795--rasahq-docs-rasa-v2.netlify.app/docs/rasa/concepts/flows#branching-conditions). I can link to them in this sentence?

Reply:
yes that works